### PR TITLE
:recycle: minor refactoring

### PR DESCRIPTION
**Pull request by koladev32**
related to #21

### CODE REVIEW

File: webhook/main.go [hunk 1]
```diff
@@ -28,23 +33,59 @@ func main() {
...
 	if err != nil {
 		logging.WebhookLogger(logging.ErrorType, fmt.Errorf("error initializing connection: %s", err))
+		log.Fatalf("error initializing connection: %v", err)
+		return
 	}
 
 	select {}
+}
+
+func createRedisClient() (*redis.Client, error) {
```
Comment (LewisYann) [inline webhook/main.go:59]: We must allow optional parameters here. We might need to instantiate Redis with a different configuration than the one in the environment

File: webhook/main.go [hunk 2]
```diff
@@ -28,23 +33,59 @@ func main() {
...
+		tlsConfig, err = redis_tls_config.CreateTLSConfig(caCertPath, clientCertPath, clientKeyPath)
+		if err != nil {
+			return nil, err
+		}
+	}
+
+	client := redis.NewClient(&redis.Options{
+		Addr:      redisAddress,
+		Password:  redisPassword,
+		DB:        0,
```
Comment (LewisYann) [inline webhook/main.go:86]: Not always, add this one to func Params with 0 as a default value

**Review by LewisYann — CHANGES_REQUESTED** [review]
Some suggestions

File: webhook/main.go (hunk 2 above)
Comment (koladev32) [reply webhook/main.go:86]: Alright

File: webhook/main.go (hunk 1 above)
Comment (koladev32) [reply webhook/main.go:59]: What can be the optional parameters ?

Comment (LewisYann) [reply webhook/main.go:59]: Host password and bd

**Pull request merged**